deploy/olm-catalog/jaeger-operator/1.14.0/jaeger-operator.v1.14.0.clusterserviceversion.yaml

```diff
@@ -94,19 +94,19 @@ spec:
     * Service dependency analysis
     * Performance / latency optimization
     * OpenTracing compatible data model
-    * Multiple storage backends: Cassandra, Elasticsearch, memory.
+    * Multiple storage backends: Cassandra, Elasticsearch, Memory, Badger.
 
     ### Operator features
 
-    * **Multiple modes** - Supports `allInOne`, `production`, and `streaming` [modes of deployment](https://github.com/jaegertracing/jaeger-operator#strategies).
+    * **Multiple modes** - Supports `allInOne`, `production`, and `streaming` [modes of deployment](https://www.jaegertracing.io/docs/latest/operator/#deployment-strategies).
 
-    * **Configuration** - Directly pass down all supported Jaeger configuration through the Operator.
+    * **Configuration** - Directly pass down all supported [Jaeger configuration](https://www.jaegertracing.io/docs/latest/operator/#configuring-the-custom-resource) through the Operator.
```

Comment:
I'm not sure what we mean by "pass down" here?  I think what we want to say is something more like "The Operator manages configuration information when installing Jaeger instances."

Reply:
Yep sounds better. Thanks